src/server/dfly_main.cc

```diff
@@ -162,6 +162,8 @@ Usage: dragonfly [FLAGS]
                   "\nbuild time: ", dfly::kBuildTime, "\n");
   };
 
+  LOG(INFO) << "Starting dragonfly " << GetVersion();
```

Comment:
please move the printing after MainInitGuard call. This call initializes properly the logging component and should be called first.

Reply:
done